### ISSUE TITLE
Scala 2.13 support

### ISSUE THREAD

**Issue by yannbolliger** [SimonJPegg/mvn_scalafmt#40]
While switching a project to Scala 2.13 I noticed that this dependency is not yet available for 2.13. The 2.13 version works for now, but it would be nice to have a 2.13 artifact.

I see from #38 that the blocker is probably `scalafmt-dynamic`. However, I wanted to asked whether support was planned in the future.

**Comment by SimonJPegg** [comment]
It is, but as you've pointed out we're dependent on `scalafmt-dynamic` and `scalafmt-core` before we can upgrade.  I'll leave this issue open as a reminder to periodically check.

**Comment by yannbolliger** [comment]
We need to wait for this issue: https://github.com/scalameta/scalafmt/issues/1434.

**Comment by yannbolliger** [comment]
This is now ready to be done: https://github.com/scalameta/scalafmt/releases/tag/v2.2.0 🎉.

**Comment by SimonJPegg** [comment]
Version `mvn-scalafmt_2.13` has been pushed and should be available shortly
